lib/filewatch/watch.rb

```diff
@@ -207,20 +207,24 @@ def subscribe(stat_interval = 1, discover_interval = 5, &block)
     end # def subscribe
 
     private
-    def expired?(stat, watched_file)
-      file_expired?(stat) && watched_file.size == stat.size
+    def close_expired?(stat, watched_file)
+      file_close?(stat) && watched_file.size == stat.size
```

Comment:
since other predicates are written in the past tense, maybe rename this to file_closed?